apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts

```diff
@@ -91,6 +99,11 @@ export class VoMembersComponent implements OnInit {
           }
           this.loading = false;
         }, err=> this.loading = false);
+
+        this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
+          this.blockManualMemberAdding = attrValue.value !== null;
+        });
```

Comment:
please move this block of code "up"
we want to load the page when all the asynchronous operations are done

Reply:
This block of code has been moved.